templates/CRM/Profile/Form/Dynamic.tpl

```diff
@@ -148,7 +148,7 @@
               {elseif $n|substr:0:5 eq 'phone'}
                 {assign var="phone_ext_field" value=$n|replace:'phone':'phone_ext'}
                 {$form.$n.html}
-                {if $form.$phone_ext_field.html}
+                {if array_key_exists$form.$phone_ext_field.html)}
```

Comment:
Should be `{if array_key_exists($phone_ext_field, $form)}` The order of params for array_key_exists is "unintuitively backwards", and also we aren't checking for the html because we need to check the phone ext member first. And then we don't actually need to check html since it's a quickform field.